clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -1226,6 +1203,34 @@ public void run() {
     }
   }
 
+  /**
+   * Create and return the directory for storing RMD schema files of this store.
+   * @return
+   * @throws IOException
+   */
+  String getRmdSchemaPath() throws IOException {
+    // make the base directory TEMP_DIR_PREFIX with 777 permissions
+    Path baseSchemaDir = new Path(TEMP_DIR_PREFIX);
```

Comment:
Can we make `TEMP_DIR_PREFIX` more specific since it is only used for RMD schema purpose?

Reply:
Sure. Will rename this variable when I re-open this PR!